### PR TITLE
Add support for url in text in DesignText

### PR DESCRIPTION
**Pull request by yiqunw700**
Ticket: #1449
No squoosh support in this pr. I find it a little bit hard to do the url annotation there so I might find some other time to look into that.

Also sort the examples in the validation so it will be easier to find a test.

### CODE REVIEW

**Comment by rylin8** [comment]
I didn't know this was something we wanted to do. Can you make a ticket for it?

File: designcompose/src/main/java/com/android/designcompose/DesignText.kt [hunk 1]
```diff
@@ -364,6 +384,29 @@ internal fun DesignText(
                 )
             } else {
                 var textLayoutResult: TextLayoutResult? = null
+                val uriHandler = LocalUriHandler.current
+                if (annotatedText.getUrlAnnotations(0, annotatedText.text.length).isNotEmpty()) {
+                    val onClick: (Int) -> Unit = { offset ->
+                        annotatedText
+                            .getUrlAnnotations(start = offset, end = offset)
```
Comment (rylin8) [inline designcompose/src/main/java/com/android/designcompose/DesignText.kt:391]: So if the hyperlink is at (10, 20) and you click at position 15, `annotatedText.getUrlAnnotation(15, 15)` will return the annotation with the hyperlink?

**Comment by yiqunw700** [comment]
> I didn't know this was something we wanted to do. Can you make a ticket for it?

Yes. Ticket https://github.com/google/automotive-design-compose/issues/1449

The existing example is the tutorial app's welcome page where it has a url link.

File: designcompose/src/main/java/com/android/designcompose/DesignText.kt (hunk 1 above)
Comment (yiqunw700) [reply designcompose/src/main/java/com/android/designcompose/DesignText.kt:391]: Yes. That is right.

**Review by rylin8 — APPROVED** [review]

**Pull request merged**